public/coda/coding_progress.js

```diff
@@ -12,14 +12,55 @@ function update_progress_ui(data) {
     }
     last_update = data["last_update"]
     document.getElementById('last_update').innerText = "Last updated: " + last_update
+    Ke_time=new Date(last_update)
+    console.log(Ke_time);
     for (var dataset_id in data["coding_progress"]) {
         var messages_count = data["coding_progress"][dataset_id]["messages_count"]
         var messages_with_label = data["coding_progress"][dataset_id]["messages_with_label"]
+        var dataset_url = document.createElement("a")
```

Comment:
let's call this dataset_link rather than dataset_url - the URL is technically just the thing starting with 'http', this is a bit more than that

Reply:
True that. Have renamed it appropriately.